InputAnswer/InputAnswer/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+namespace InputAnswer
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int excitement = 0;
```

Comment:
Style: I'm a fan of declaring variables when they will be used, rather than before - this is because if there were 50 lines of code between the declaration and the use, I might not instantly remember what it was for/where it was set.